src/libraries/Microsoft.PowerFx.LanguageServerProtocol/JsonRpc/JsonRpcHelper.cs

```diff
@@ -34,7 +34,13 @@ public enum ErrorCode
             code = (int)code
         });
 
-        public static string CreateErrorResult(string id, object error) => JsonSerializer.Serialize(
+        public static string CreateErrorResult(string id, ErrorCode code, string message) => CreateErrorResult(id, new
+        {
+            code = (int)code,
+            message = message
+        });
+
+        private static string CreateErrorResult(string id, object error) => JsonSerializer.Serialize(
```

Comment:
Made private - go through the helper on line 37. 
We had a bug passing in the wrong object.